content/api/commands/intercept.md

```diff
@@ -1326,30 +1315,33 @@ from propagating to the next matching response handler in line. See
 
 ## `StaticResponse` objects
 
-A `StaticResponse` represents a stubbed response to an HTTP request. You can
-supply a `StaticResponse` to Cypress in 3 ways:
+A `StaticResponse` represents a
+[statically defined response (stub)](#Stubbing-a-response).
+
+The following properties are available on `StaticResponse`.
+
+| Option            | Description                                                                                                                                                                                                                       |
+| ----------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| statusCode        | HTTP response status code                                                                                                                                                                                                         |
+| headers           | HTTP response headers                                                                                                                                                                                                             |
+| body              | Serve a static response body (`object`, `string`, `ArrayBuffer`) (when `fixture` is omitted).                                                                                                                                     |
+| fixture           | Serve a fixture as the HTTP response body (allowed when `body` is omitted). Read the contents with an encoding other than the [default for the fie type](/api/commands//fixture#Encoding), pass the fixture like `path,encoding`. |
```

Comment:
this should read 'file type', right? Also, some rogue forward slashes in the slug which will be problematic.

Reply:
@jaffrepaul yes good eye